public/static/partials/user_host_terminate.html

```diff
@@ -3,8 +3,14 @@
     <span class="semi-muted">
       Terminate <strong>[[curHostData.distro._id]]</strong> host <strong>[[concatName(curHostData.id, curHostData.display_name)]]</strong>?
 	</span>
+    <div ng-show="curHostData.no_expiration" style="margin-bottom:5px">
+      <span class="semi-muted">
+        Host is unexpirable. If you'd really like to terminate, please type "delete":
+      </span>
+      <input type="text" id="input-type"  ng-model="checkDelete"/>
+    </div>
     <br>
     <button type="button" class="btn btn-danger" style="float: right;" data-dismiss="modal">Cancel</button>
-    <button type="button" class="btn btn-primary" style="float: right; margin-right: 10px;" ng-click="updateHostStatus('terminate')">Yes</button>
+    <button type="button" class="btn btn-primary" style="float: right; margin-right: 10px;" ng-disabled="invalidDelete(checkDelete)" ng-click="updateHostStatus('terminate')">Yes</button>
```

Comment:
I'm not sure what you mean by enter handler 🤔 the submit button is disabled though

Reply:
when we open the modal we [register a handler](https://github.com/evergreen-ci/evergreen/blob/198576251751530106869b7d8b605de2c4612da4/public/static/js/spawned_hosts.js#L861) for the enter key that calls `updateHostStatus` with the action. If the user hits enter instead of clicking the submit button it looks like we will not check the user has confirmed.

Reply:
Oh cool! I think I'm just going to remove the enter handler, since this module potentially has information now (it looks like we only use enter handlers for things without information; for example Spawn Host doesn't have one).